aria-practices.html

```diff
@@ -710,100 +710,102 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
-        <li>It is displayed if the value of the textbox is altered in a way that creates one or more partial matches to a suggested value.</li>
+        <li>It is displayed when the <kbd>Down Arrow</kbd> key is pressed or the <q>Open</q> button is activated. Optionally, if the combobox is editable and contains a string that does not match an allowed value, expansion does not occur.</li>
+        <li>It is displayed when the combobox receives focus even if the combobox is editable and empty.</li>
+        <li>If the combobox is editable, the popup is displayed only if a certain number of characters are typed in the combobox and those characters match some portion of one of the suggested values.</li>
+        <li>If the combobox is editable, it is displayed if the value of the combobox is altered in a way that creates one or more partial matches to a suggested value.</li>
       </ul>
-      <p>Combobox widgets are useful for setting the value of a single-line textbox in one of two types of scenarios:</p>
+      <p>Combobox widgets are useful for acquiring user input in either of two scenarios:</p>
       <ol>
         <li>
-          The value for the textbox must be chosen from a predefined set of allowed values,
-          e.g., a location field must contain a valid location name.
+          The value must be one of a predefined set of allowed values, e.g., a location field must contain a valid location name.
           Note that the listbox and menu button patterns are also useful in this scenario; differences between combobox and alternative patterns are described below.
         </li>
         <li>
-          The textbox may contain any arbitrary value, but it is advantageous to suggest possible values to the user,
-          e.g., a search field may suggest similar or previous searches to save the user time.
+          An arbitrary value is allowed, but it is advantageous to suggest possible values to users.
+          For example, a search field may suggest similar or previous searches to save the user time.
         </li>
       </ol>
       <p>
-        The nature of the suggested values and the way the suggestions are presented is called the autocomplete behavior.
+        The nature of possible values presented by a popup and the way they are presented is called the autocomplete behavior.
         Comboboxes can have one of four forms of autocomplete:
       </p>
       <ol>
         <li>
-          <strong>No autocomplete:</strong> When the popup is triggered, the suggested values it contains are the same regardless of the characters typed in the textbox.
+          <strong>No autocomplete:</strong> The combobox is editable, and  when the popup is triggered, the suggested values it contains are the same regardless of the characters typed in the combobox.
           For example, the popup suggests a set of recently entered values, and the suggestions  do not change as the user types.
```

Comment:
```suggestion
          For example, the popup suggests a set of recently entered values, and the suggestions do not change as the user types.
```
Remove space after "suggestions."